network-store-model/src/main/java/com/powsybl/network/store/model/RegulationPointAttributes.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.model;
+
+import com.powsybl.iidm.network.IdentifiableType;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@Schema(description = "Regulation Point attributes")
+public class RegulationPointAttributes extends AbstractAttributes implements Attributes {
+
+    @Schema(description = "Regulated Equipment Id")
+    private String regulatedEquipmentId;
+
+    @Schema(description = "Identifiable type")
+    private IdentifiableType identifiableType;
+
+    @Schema(description = "Local terminal")
+    private TerminalRefAttributes localTerminal;
+
+    @Schema(description = "Regulating terminal")
+    private TerminalRefAttributes regulatingTerminal;
+
+    @Schema(description = "Regulation mode ordinal")
+    private int regulationMode;
```

Comment:
maybe it should be a string for db migration ?

Reply:
Yes, it's better to keep the enum here ...

You have to do a PR in network-store-server, for the changes in the column mappings and the migration in the database ...

Reply:
Yes I'm trying to do it
I dont think we can keep the enum because there are a regulation mode for generators, twt ...